Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayer.h

```diff
@@ -163,6 +164,11 @@ namespace Multiplayer
         //! @param handler The NoServerLevelLoadedEvent handler to add
         virtual void AddNoServerLevelLoadedHandler(NoServerLevelLoadedEvent::Handler& handler) = 0;
 
+        //! Adds a VersionMismatchEvent Handler which is invoked whenever a connection is made between 2 apps that have differing multiplayer components.
```

Comment:
We should be clear with the language here. If this can detect version mismatches between any two MP endpoints, that's the language we should go with. Granted we only support Client/Server presently.

Reply:
updated!